YogaKit/YGLayout.m

```diff
@@ -318,6 +312,14 @@ static void YGAttachNodesFromViewHierachy(UIView *const view)
   if (yoga.isLeaf) {
     YGRemoveAllChildren(node);
     YGNodeSetMeasureFunc(node, YGMeasureView);
+
+    YGSize oldSize = yoga.cachedSize;
+    YGSize newSize = YGMeasureView(node, 0, YGMeasureModeUndefined, 0, YGMeasureModeUndefined);
```

Comment:
I'm going to be honest, I'm not sure this is something we would want to do.

1. We are now guaranteeing that every leaf node is going to be measure during every single layout call, that's going to get pretty expensive, especially if we have to size text.

2. Are viewed measured with zero size supposed to return a non-zero value? I don't think that's an assumption we should make. As a result, I'm not sure that the result is something that we should be comparing to an *actual* measurement.